masterlist.yaml

```diff
@@ -443,7 +405,7 @@ globals:
     subs:
       - '**[Place Everywhere](https://www.nexusmods.com/fallout4/mods/9424)**'
       - 'Fallout 4'
-    condition: 'file("F4SE/Plugins/place.dll") and file("../f4se_loader.exe") and ((version("F4SE/Plugins/place.dll", "1.18.0.1163", ==) and version("../Fallout4.exe", "1.10.163.0", !=)) or (version("F4SE/Plugins/place.dll", "1.18.0.1163", <) and version("../Fallout4.exe", "1.10.163.0", ==)))'
+    condition: 'file("F4SE/Plugins/place.dll") and file("../f4sevr_loader.exe") and ((version("F4SE/Plugins/place.dll", "1.18.0.1163", ==) and version("../Fallout4.exe", "1.10.163.0", !=)) or (version("F4SE/Plugins/place.dll", "1.18.0.1163", <) and version("../Fallout4.exe", "1.10.163.0", ==)))'
```

Comment:
In case you missed it, this message is still checking the version of `Fallout.exe` instead of `FalloutVR.exe`. Not sure if it's documented anywhere what VR runtimes PE is compatible with or if removing the message altogether would be more appropriate.

Reply:
> In case you missed it, this message is still checking the version of `Fallout.exe` instead of `FalloutVR.exe`. Not sure if it's documented anywhere what VR runtimes PE is compatible with or if removing the message altogether would be more appropriate.

I ended up removing it, and all of the other FO4 executable checks.